back-nest/Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 COPY package*.json ./
 
-RUN npm install
+RUN npm install --force
```

Comment:
Try to resolve your dependencies and be able to install them without `force` flag